### ISSUE TITLE
Build failed - 'Makefile:284: recipe for target 'release/bin/monero-wallet-gui' failed'

### ISSUE THREAD

**Issue by 1337tester** [monero-project/monero-gui#1186]
The recent build is not building on my Ubuntu Linux - [log_build.txt](https://github.com/monero-project/monero-gui/files/1822181/log_build.txt)

OS - Ubuntu 17.10, GNOME
commit -  da0155e

**Comment by 1337tester** [comment]
Just to spare some time here, I have all the dependencies (made double-sure) and I did builds of course with previous versions successfully

**Comment by radfish** [comment]
`src/wallet/api/CMakeFiles/obj_wallet_api.dir/build.make:62: recipe for target 'src/wallet/api/CMakeFiles/obj_wallet_api.dir/wallet.cpp.o' failed`

so this might be an issue for monero not monero-gui but I don't see the error in the log. Does monero build for you?

**Comment by 1337tester** [comment]
yes, the newest build (yesterday from master) of monero cli builds quite smoothly

**Comment by qubenix** [comment]
Seems that I have the same issue on Debian 9: [build.log](http://termbin.com/f04m). Monero builds no problem from current master.

**Comment by qubenix** [comment]
Some strangeness, on build attempts 2 and 3 I get a new error, different error than the first attempt. This is the log from subsequent build attempts: http://termbin.com/b7ms.

**Comment by qubenix** [comment]
#1172 fixes this for me.

**Comment by 1337tester** [comment]
Is this already in the master? I tried the newest master (merge of PR #1144) just now - still the same error

**Comment by 1337tester** [comment]
@qubenix  - yes, tried yesterday as #1172 was merged and it can be built

-> can be closed
